docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  app:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    environment:
+      - DEBUG=False
```

Comment:
So... `docker-compose` has some interesting ideas of where environment variables should come from, so I just explicitly set it here.

See: https://docs.docker.com/compose/environment-variables/#the-env-file